src/components/AutoSuggest/components/SuggestList/SuggestList.tsx

```diff
@@ -98,7 +99,23 @@ const SuggestList: FC<SuggestListProps> = ({
     [onSelect]
   )
 
-  if (!options.length) {
+  if (options.length === 0) {
+    options = [
+      {
```

Comment:
Dacht eraan data optional te maken, dat is alleen te ' permissive ' voor een aantal plekken, dus leek dit de minste schade te doen.